packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -130,13 +130,19 @@ function Thread( {
 		setShowCommentBoard( false );
 	};
 
+	const replies = thread?.reply || [];
```

Comment:
I think `thread.reply` will always be an array, that's how they're initialized.

https://github.com/WordPress/gutenberg/blob/0d5b7f080ec7a5f4cdeca5150c41c4ecaa71d3f2/packages/editor/src/components/collab-sidebar/hooks.js#L49